docs/en/concepts-and-designs/OAP-Cluster-Management.md

```diff
@@ -0,0 +1,39 @@
+# Observability Analysis Platform Cluster Management
+OAP(Observability Analysis Platform) server is a distributed system, services need to find each 
+other. i.e. a web service needs to find query service, level 1 aggregate service need to find 
+level 2 aggregate service. Cluster management is just a client-side implementation. It must work 
+together with a distributed coordination server, (i.e. Zookeeper, Consul, Kubernetes.) unless 
+running in standalone mode.
+
+## Architecture overview
+<img src="https://skywalkingtest.github.io/page-resources/6.0.0-alpha/cluster_management.png"/>
+
+### Cluster Management Plugins
+By default, OAP server provides two implementations for cluster management, which are standalone 
+and zookeeper. When the applications being monitored are small, you can choose the standalone mode.
+If they are big, you must choose the cluster mode by zookeeper plugin, or you can implement another 
```

Comment:
1. `If they are big` - > `Otherwise`
1. It(Scale) is big.
1. Remove ~~another~~